docs/options/model-transports.md

```
@@ -9,6 +9,7 @@
 | rlcr          | rlcr://   | [`ramalama.com`](https://registry.ramalama.com) |
 | OCI Container Registries | oci://, docker:// | [`opencontainers.org`](https://opencontainers.org)||||Examples: [`quay.io`](https://quay.io),  [`Docker Hub`](https://docker.io),[`Artifactory`](https://artifactory.com)|
 | Hosted API Providers | openai:// | [`api.openai.com`](https://api.openai.com)|
+|                      | orcarouter:// | [OrcaRouter](https://www.orcarouter.ai) — `https://api.orcarouter.ai/v1` |
 Models can be specified using a shortname (e.g. `tiny`) which is resolved via `shortnames.conf`, or with an explicit transport prefix such as `huggingface://`, `oci://`, `ollama://`, `https://`, `http://`, or `file://`. Models in the `<org>/<model>` format without a prefix are pulled from Hugging Face.
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)